.ci/Jenkinsfile

```diff
@@ -445,26 +446,90 @@ stage('Build') {
     }
     finally {
         stage('Merge Artifacts') {
-            node(pCloud) {
-                checkout scm  // checking out the SCM so the coverage report can load the source
-                builds.each { item ->
-                    copyArtifacts(
-                        projectName: item.fullProjectName,
-                        selector: specific("${item.number}"),
-                        fingerprintArtifacts: true,
-                        optional: true,
-                    )
+            String yaml = """
+apiVersion: v1
```

Comment:
[nit] Not a huge fan of embedding YAML's in functional code like this.  Could we make this a config YAML job template somewhere, read it in and populate the required config fields before passing it to `podArgs`?

Reply:
Unfortunately, for security reasons, we can't easily. If the yaml is in a separate file, then external contributors could modify the yaml to create arbitrary pods on our cluster. Changes to the Jenkinsfile itself are protected, though -- Jenkins won't build PRs that modify the Jenkinsfile unless the commit author is from Mosaic (validated via the GH api, I presume).